modules/virt-installing-virt-operator.adoc

```diff
@@ -37,7 +35,12 @@ You can install the {VirtProductName} Operator from the {product-title} web cons
 Attempting to install the {VirtProductName} Operator in a namespace other than `openshift-cnv` causes the installation to fail.
 ====
 
-.. For *Approval Strategy*, ensure that *Automatic*, which is the default value, is selected. {VirtProductName} automatically updates when a new z-stream release is available.
+.. For *Approval Strategy*, you can select either *Automatic* or *Manual*. It is highly recommended that you select *Automatic*, which is the default value, so that {VirtProductName} automatically updates when a new version is available in the *stable* update channel. If you use the *Manual* approval strategy, you must manually approve all pending updates.
```

Comment:
This change incldues more details about manual, but to me the wording got softened.

"Manual" should only be used as an expcetion.

Reply:
@fabiand Please review the updated wording. Thanks!